signingscript/src/signingscript/sign.py

```diff
@@ -537,22 +537,24 @@ def _get_mac_sigpath(from_):
 
 # _get_widevine_signing_files {{{1
 def _get_widevine_signing_files(file_list):
-    """Return a dict of path:signing_format for each path to be signed."""
+    """Return a dict of path:is_blessed for each path to be signed."""
     files = {}
     for filename in file_list:
-        fmt = None
         base_filename = os.path.basename(filename)
+        if base_filename not in _WIDEVINE_BLESSED_FILENAMES and base_filename not in _WIDEVINE_NONBLESSED_FILENAMES:
+            log.debug(f"{filename} is not a widevine signing file")
+            continue
+
+        blessed = False
         if base_filename in _WIDEVINE_BLESSED_FILENAMES:
-            fmt = "widevine_blessed"
-        elif base_filename in _WIDEVINE_NONBLESSED_FILENAMES:
-            fmt = "widevine"
-        if fmt:
-            log.debug("Found {} to sign {}".format(filename, fmt))
-            sigpath = _get_mac_sigpath(filename)
-            if sigpath not in file_list:
-                files[filename] = fmt
-            else:
-                log.debug("{} is already signed! Skipping...".format(filename))
+            blessed = True
+
+        log.debug("Found {} to sign {}".format(filename, blessed))
```

Comment:
"Found firefox/firefox-bin to sign False" is a bit of an odd message

Reply:
You're right, this could be improved.